src/Program.cs

```diff
@@ -8,13 +9,18 @@
 using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
+using Microsoft.Diagnostics.NETCore.Client;
 using Microsoft.Diagnostics.Runtime;
 
 namespace ClrStack
 {
     internal static class Program
     {
         private const string ThreadDumpDirEnvVar = "THREAD_DUMP_DIR";
+        private const string NoSuspend = "--no-suspend";
+        private const string Timeout = "--timeout=";
+        private const string DumpProcessTo = "--dumpProcessTo";
+        private const string CreateSnapshotAndAttach = "--createSnapshotAndAttach";
```

Comment:
Mind the conventions ☝️ 
```suggestion
        private const string DumpProcessTo = "--dump-process-to";
        private const string CreateSnapshotAndAttach = "--create-snapshot-and-attach";
```